### DISCUSSION TITLE
PCAPs for Suricata alerts

### DISCUSSION THREAD

**Discussion by vijaytejak** [Security-Onion-Solutions/securityonion#5125] [Unsupported Versions] [answered]
Hi all, I have set up Security Onion, as the disk is filling very fast, I have disabled the Stenographer, I want the PCAPs for all the alerts that came in the Alerts section like for each Suricata alerts i want complete PCAP for further analysis.

**Comment by TOoSmOotH** [comment, marked as answer]
If you want PCAP then you have to enable Steno. There is no alert only option for PCAP. Suricata alerts include the packet that tripped the rule in the alert record.
